deploy/kubernetes/controller.yaml

```diff
@@ -238,16 +252,25 @@ spec:
               mountPath: /var/lib/csi/sockets/pluginproxy/
         - name: csi-snapshotter
           image: quay.io/k8scsi/csi-snapshotter:v1.0.1
+          imagePullPolicy: Always
```

Comment:
We might think about changing these policies to `IfNotPresent` so the whole deployment doesn't fail if `quay.io` can't be reached.

Reply:
👍 for all the tagged images, for `amazon/aws-ebs-csi-driver:latest` we should just remove `imagePullPolicy` and let Kubernetes do its thing and try to pull it since it has the `:latest` tag.